Makefile

```diff
@@ -140,8 +141,8 @@ protoc-gen-all: protoc-gen-code protoc-gen-docs
 # Generate protobuf go code
 protoc-gen-code:
 	@protoc \
-		--go_out=plugins=grpc:./rpc \
-		--go_opt=module=github.com/stripe/stripe-cli/rpc \
+		--go-grpc_out=require_unimplemented_servers=false,=plugin=grpc:./rpc \
+		--go-grpc_opt=module=github.com/stripe/stripe-cli/rpc \
```

Comment:
You need all four flags, please add the two deleted ones back in.

In older versions, go_out and go_opt would generate both the protobuf messages and the gRPC services, but now they are generated separately and each require their own flags.

Reply:
Let me try again, when I was running the command it told me the flags didn't exist.

Reply:
@vcheung-stripe yea it doesn't work:
```
make protoc-gen-all
--go_out: protoc-gen-go: plugins are not supported; use 'protoc --go-grpc_out=...' to generate gRPC

See https://grpc.io/docs/languages/go/quickstart/#regenerate-grpc-code for more information.
```